templates/taxi/car_detail.html

```diff
@@ -12,6 +12,18 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+
+  {% csrf_token %}
+  {% if user in car.drivers.all %}
+    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-danger link-to-page">
+      Delete me from this car
+    </a>
+  {% else %}
+    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-success link-to-page">
+      Assign me to this car
+    </a>
+  {% endif %}
```

Comment:
You can use tag `{% with %}` to not repeat yourself and implement this using only 1 `<a>` tag